_topnav_mobile.php

```diff
@@ -0,0 +1,20 @@
+<nav class="topnav-mobile">
+  <div class="navbar">
+    <a href="#" data-toggle=".topnav-mobile"><i class="fa fa-bars fa-lg"></i></a>
+    <div class="logo-wrapper">
+      <a href="<?php bloginfo('url'); ?>" class="custom-logo-link"><img src="<?php echo get_template_directory_uri(); ?>/images/logo-mobile.png"></a>
```

Comment:
This doesn't work, because it doesn't allow us to modify the container size to effect the image size inside it.  The logo will have to be hardcoded (which it already is on master)